benchalerts/tests/integration_tests/test_talk_to_conbench_integration.py

```diff
@@ -54,9 +54,10 @@
 def test_get_comparison_to_baseline(
     monkeypatch: pytest.MonkeyPatch, conbench_url, commit, expected_len, expected_bip
 ):
-    pytest.skip(
-        "https://github.com/conbench/conbench/issues/745 means timeouts cause this to fail"
-    )
+    if "ursa" in conbench_url:
```

Comment:
The velox end-to-end tests work now because of #766 . This partially addresses #748 (though the arrow end-to-end tests are still failing)